ftp.go

```diff
@@ -191,20 +195,24 @@ func (c *ServerConn) pasv() (port int, err error) {
 	start := strings.Index(line, "(")
 	end := strings.LastIndex(line, ")")
 	if start == -1 || end == -1 {
-		err = errors.New("Invalid PASV response format")
-		return
+		goto format_error
```

Comment:
Code style suggestion.

Replace all instances of `goto format_error` with:

```Go
return 0, errors.New("Invalid PASV response format")
```

It's much cleaner, more readable and idiomatic.